lib/screens/search_screen/bloc/github_search_bloc.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:top_git_rep/models/search_result.dart';
+import 'package:top_git_rep/services/git_api_provider.dart';
+
+import 'github_search_event.dart';
+import 'github_search_state.dart';
+
+class GithubSearchBloc extends Bloc<GithubSearchEvent, GithubSearchState> {
+  GithubSearchBloc({required this.gitProvider})
+      : super(const GithubSearchState());
+
+  static const List<String> languages = [
```

Comment:
I'd like to see there a todo, which would describe why there're hardcoded strings